src/libcore/sync/atomic.rs

```diff
@@ -1119,8 +1123,14 @@ macro_rules! atomic_int {
 
         #[$stable_from]
         impl From<$int_type> for $atomic_type {
-            #[inline]
-            fn from(v: $int_type) -> Self { Self::new(v) }
+            doc_comment! {
+                concat!("Converts a `", $int_type, "` into an `", $atomic_type, "`.
```

Comment:
The `concat!` macro doesn't like these idents in here. We might have to drop back to a generic doc comment like _Converts an integer into an atomic integer._

Reply:
Thanks @KodrAus wonder if `stringify!($atomic_type)` would work like a few lines below:

```
"use std::sync::atomic::", stringify!($atomic_type), ";"
```